common/lib/xmodule/xmodule/modulestore/tests/django_utils.py

```diff
@@ -28,7 +29,6 @@
 from xmodule.modulestore.tests.factories import XMODULE_FACTORY_LOCK
 from xmodule.modulestore.tests.mongo_connection import MONGO_HOST, MONGO_PORT_NUM
 
-
```

Comment:
Note spurious line removal here, in case this ends up getting backported.

Reply:
(causes pep8 failure, fixed in https://github.com/edx/edx-platform/pull/29199)

Reply:
Thank you. Sorry I missed this.

Reply:
The status check wasn't required, which made it easy to miss. :-) We've fixed that, now.